openslides/motions/models.py

```diff
@@ -339,7 +339,11 @@ def get_new_version(self, **kwargs):
         attributes, it is populated with the data from the last version
         object if such object exists.
         """
-        new_version = MotionVersion(motion=self, **kwargs)
+        if self.pk is None:
+            # Do not reference the MotionVersion object to an unsaved motion
```

Comment:
```python
str(Motion())
```
This happens (for example) if you call get_title() from a Motion without a Version. See get_last_version() and get_active_version()